doc/en/skipping.rst

```diff
@@ -247,9 +247,20 @@ imperatively:
         if not valid_config():
             pytest.xfail("failing configuration (but should work)")
 
-This will unconditionally make ``test_function`` ``XFAIL``. Note that no other code is executed
-after ``pytest.xfail`` call, differently from the marker. That's because it is implemented
-internally by raising a known exception.
+.. code-block:: python
+
+    def test_function2():
+        import slow_module
+
+        if slow_module.slow_function():
+            pytest.xfail("slow_module taking too long")
+
+These two examples illustrate situations where you don't want to check for a condition
+at the module level, which is when a condition would otherwise be evaluated for marks.
+
+They will unconditionally make the two examples ``XFAIL``. Note that no other code is executed
```

Comment:
I know this is from before, but I'm confused by the "unconditionally" here -- the `pytets.xfail` is literally under an `if`. Maybe I'm misunderstanding?

Reply:
I was confused by it as well, but didn't want to remove without proper justification. I'll update with the "unconditionally" removed, and leave it to you for approval.

Reply:
Updated PR based on feedback